test/parallel/test-http-malformed-request.js

```diff
@@ -50,7 +50,7 @@ server.on('listening', function() {
     c.end();
   });
 });
-
+countdown.dec()
 process.on('exit', function() {
```

Comment:
@AyoAlfonso No need of `process.on('exit')` now. Also, please run this tests locally first because I don't think this is passing since, `nrequests_completed` is not defined anymore.